src/routes/expenses.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
```

Comment:
Consider removing 'use strict' in this file since, with ES6+ syntax, you don't need it anymore. The script will be in strict mode by default 🚀